Docs/developer_guide/build_instructions/linux.md

```diff
@@ -76,6 +76,41 @@ sudo apt update && sudo apt install git subversion build-essential cmake cmake-c
   qtbase5-private-dev libqt5x11extras5-dev libxt-dev
 ```
 
+### Ubuntu 23.04 (Lunar Lobster)
+
+:::{warning}
+If you get cmake version error, download build and install cmake 3.25.3:
```

Comment:
I like the idea of getting people the right CMake is good - maybe these instructions below in a separate section about CMake that applies to more than just this ubuntu version, perhaps with notes about how to install the latest CMake on multiple OSes.